package.json

```diff
@@ -22,15 +22,17 @@
     "core"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=14"
   },
   "main": "dist/sn-table.js",
   "scripts": {
-    "build": "rollup -c rollup.config.js && rollup -c rollup-core.config.js && shx cp assets/* dist",
+    "build": "yarn run locale:generate && rollup -c rollup.config.js && rollup -c rollup-core.config.js && shx cp assets/* dist",
     "build:watch": "rollup -c rollup.config.js -w",
     "lint": "eslint src",
     "start": "nebula serve --build false --type sn-table",
     "sense": "nebula sense",
+    "locale:verify": "node locale/tools/verify-translations.js",
```

Comment:
Verify is not used anywhere? Do we only want this available on local environments? Guess it should be included in the pipeline as well

Reply:
Yes, it should. Done.